### PR TITLE
Hdf5_save addded

### PR DESCRIPTION
**Pull request by KasperPRasmussen**
Hdf5_save function added.

PyCharm also did automatic PEP8 coding style edits.

### CODE REVIEW

File: src/anypytools/abcutils.py [hunk 1]
```diff
@@ -443,6 +444,43 @@ def save_results(self, filename, append=False):
         else:
             raise ValueError('Noting to save')
 
+    def save_to_hdf5(self, filename: str, batch_name: str):
+        if self.cached_tasklist:
+            any_output = [task.get_output() for task in self.cached_tasklist]
+            any_output = AnyPyProcessOutputList(any_output)
+            with h5py.File(filename, "w") as f:  # , compression="gzip" # Not sure how much gzip effects reading speed.
+                group = f.create_group(batch_name)
+                task_names = []
```
Comment (melund) [inline src/anypytools/abcutils.py]: Det kunne være skrevet kort: `task_names = [ r['task_name'] for r in any_output]`

**Comment by melund** [comment]
Thank you Kasper. I will go ahead and merge it. A few notes for the future. Try to keep different changes to different commits. Not a big deal, though. Also, there is a lot repeated code in the save_to_hdf5 function. We can always change that later.

**Comment by melund** [comment]
Argg.. Crap. Just realized, you added function annotations. I don't think they are supported in python 2.7. I will just make a commit to remove them again

**Comment by msan00** [comment]
Hej Morten

Hyggeligt at følge jeres udviklingsdialoger, men hvordan slår man disse email notifikationer fra?

Hilsen
Michael



Sent from my Samsung Galaxy smartphone.


-------- Original message --------

**Comment by melund** [comment]
Hej  Skipper ( @msan00 ). Gå ind på https://github.com/settings/notifications og vælg fjern afkrydsningen i 'email' felterne...

**Comment by melund** [comment]
@KasperPRasmussen . Nu har jeg rettet function annoteringen, og bygget en ny udgave af AnyPyTools med din hd5 funktionalitet. Den kan installeres via conda på sædvanlig vis.

**Pull request merged**